cors.go

```diff
@@ -0,0 +1,65 @@
+// Package cors is net/http handler to handle CORS related requests
+// as defined by http://www.w3.org/TR/cors/
+package cors
+
+import (
+	"net/http"
+
+	"github.com/rs/cors"
+)
+
+// Options is a configuration container to setup the CORS middleware.
+type Options struct {
+	// AllowedOrigins is a list of origins a cross-domain request can be executed from.
+	// If the special "*" value is present in the list, all origins will be allowed.
+	// An origin may contain a wildcard (*) to replace 0 or more characters
+	// (i.e.: http://*.domain.com). Usage of wildcards implies a small performance penality.
+	// Only one wildcard can be used per origin.
+	// Default value is ["*"]
```

Comment:
Seems like if you don't define any `AllowedOrigins`, the default behavior will be allow any origin, which is analog to `*`. I was wrong about this. You can always define `AllowedOrigins: ["*"]` if you want it explicitly.